vendor.conf

```diff
@@ -26,7 +26,7 @@ github.com/imdario/mergo v0.3.6
 golang.org/x/sync 1d60e4601c6fd243af51cc01ddf169918a5407ca
 
 # buildkit
-github.com/moby/buildkit 8cf9bec86a7f11fe6591804aee152c8e8a7a8a0d # v0.3.3
+github.com/moby/buildkit d9f75920678e35090025bb89344c5370e2efc8e7
```

Comment:
Will this one be tagged as v0.3.4, or are more changes incoming before that?

Reply:
There is only one fix that matters in here so not planning for `v0.3.4` atm.